koku/reporting/migrations/0116_partitioned_ocplineitemusage_daily_summary.py

```diff
@@ -0,0 +1,160 @@
+# Generated by Django 2.2.13 on 2020-06-17 22:54
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("api", "0021_db_functions"), ("reporting", "0115_populate_source_uuid")]
+
+    operations = [
+        migrations.RunSQL(
+            """
+-- ===================================
+--  Create tracking table
+-- ===================================
+
+CREATE TABLE IF NOT EXISTS partitioned_tables (
```

Comment:
I thikn the partitioned_tables table could be built from a Django model since it's okay for it to manage this.

Reply:
Yes.

Reply:
done